openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -0,0 +1,19 @@
+"""
+Transformers for all the enrollment events
+"""
+
+
+def edx_course_enrollment_deactivated(current_event, caliper_event):
+    """
+    Caliper specific log for edx course unenrollment
+    """
+    caliper_event['type'] = 'Enrollment'
+    caliper_event['actor']['type'] = 'Person'
+    caliper_event['object'] = dict()
+    caliper_event['object'].update({
+        "course_id": caliper_event['course_id'],
+        "user_id": caliper_event['user_id'],
+        "mode": current_event['event']['mode']
+    })
+    caliper_event['action'] = 'Deactivated'
+    return caliper_event
```

Comment:
this can be a little more simpler 

```
object = {
    "course_id": caliper_event['course_id'],
    "user_id": caliper_event['user_id'],
    "mode": current_event['event']['mode']
}

caliper_event.update({
    'object' = object,
    'type' = 'Enrollment',
    'action' = 'Deactivated'
})
```

Reply:
Done